src/google.ts

```diff
@@ -96,9 +104,32 @@ export function createGoogleClient(
     }
   }
 
+  /**
+   *
+   * A function to add a member to the beta group.
+   *
+   * @param email The email of the user you want to add to the beta group
+   */
+  async function addMemberToBetaGroup(email: string) {
+    try {
```

Comment:
you can use trycatch https://github.com/gcanti/fp-ts/blob/master/src/Either.ts#L518